plat/mediatek/mt6795/include/plat_macros.S

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
So far mt6795 dump the same registers with  mt8173 while crash report generates. In later SoC, GiCv3 related registers will be dumped.